pom.xml

```diff
@@ -275,7 +275,19 @@
                    </differences>
                  </revapi.differences>
             -->
-            
+            <revapi.differences>
+              <justification>AspectJ false positives</justification>
```

Comment:
I'd explain it as "Not a breakage. We fix the backward compatibility issue by adding back the removed code using an Aspect".

Reply:
This error has nothing to do with the retro compatibility we implement. It basically just complain that the aspect is not `privileged` anymore (since the concept does not really exist in the Java version of AspectJ).

Reply:
hmm ok, I'm surprised that revapi would know about aspects, and it's supposed to execute only on bytecode (where "privileged" doesn't exist.

Reply:
idea: We could add it to our list of annotations that should be ignore by revapi

Reply:
I guess ignoring the whole `org.aspectj.internal.*` might make sense.